src/main/java/com/easypost/model/Tracker.java

```diff
@@ -148,7 +149,16 @@ public static boolean createList(Map<String, Object> params) throws EasyPostExce
 	public static boolean createList(Map<String, Object> params, String apiKey) throws EasyPostException {
 		String createListUrl = String.format("%s/create_list", classURL(Tracker.class));
 
-		request(RequestMethod.POST, createListUrl, params, Object.class, apiKey);
+		int count = 0;
+		Map<String, Object> newParams = new HashMap<String, Object>();
+		Map<String, Object> trackers = new HashMap<String, Object>();
+		for (Object tracker : (ArrayList)params.get("trackers")) {
```

Comment:
Should there be a space when type-casting `(ArrayList)params...` -> `(ArrayList) params...` ?

Reply:
We can run a linter on the entire library to clean this up in the future.

Reply:
New `indent_cleanup` branch for this: https://github.com/EasyPost/easypost-java/tree/indent_cleanup